bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -164,7 +164,7 @@ public async Task<IActionResult> Login(string returnUrl)
             }
             else
             {
-                throw new Exception("No domain_hint provided.");
+                throw new Exception(_i18nService.GetLocalizedHtmlString("NoDomainHintProvided"));
```

Comment:
for an exception message, should just use `.Translate("key")` or `.T("key")`, vs. `GetLocalizedHtmlString("key")`

Reply:
corrected